adlfs/tests/conftest.py

```diff
@@ -59,7 +59,7 @@ def storage(host):
 
 @pytest.fixture(scope="session")
 def event_loop():
-    loop = asyncio.get_event_loop()
```

Comment:
`get_event_loop()` is deprecated in 3.10, and separate sessions should use their own loops